src/test/scala/org/constellation/primitives/CheckpointBlockValidatorNelTest.scala

```diff
@@ -433,7 +433,7 @@ class ValidationSpec
 
   "two groups of checkpoint blocks lower blocks beyond the snapshot" when {
     "first group is internally inconsistent" should {
-      "not pass validation" in {
+      "not pass validation" ignore { // TODO: wkoszycki #420 enable recursive validation
```

Comment:
What's the problem with that test? It's working both on dev and my local branch so I assume there is sth wrong with your's changes

Reply:
Current test is not properly implemented it works because all data is stored, anyway I have added the data as it is now and moved this `TODO` down